Makefile

```diff
@@ -757,7 +757,7 @@ endif
 ## Enable passthrough of a PCI device in QEMU by passing its slot information to VFIO.
 ## Slot information is its bus, device, and function number assigned by the host OS, e.g., 'vfio=59:00.0'.
 ifdef vfio
-	QEMU_FLAGS += -device vfio-pci,host=$(vfio)
+	QEMU_FLAGS += -device vfio-pci,host=$(vfio),romfile=
```

Comment:
So initially when I used pci passthrough for the ixgbe device, QEMU was always trying to boot via pxe first and that added 5-10 seconds in the boot time.
This was just a workaround I found [here](https://askubuntu.com/questions/190929/how-do-i-disable-unwanted-ipxe-boot-attempt-in-libvirt-qemu-kvm) that prevented that.

I've removed it now because now it's booting from the cd rather than through pxe first, even without this flag.
If it happens again, I can submit another PR. There may be a better workaround such as specifying the boot order.

Reply:
I don't mind leaving it in or adding it back later, so long as we document it. Mostly, I was wondering whether it should *always* be used, or just with the vfio arg.

Reply:
I think just with the vfio arg, because it's specifying something about that particular device.